synapse/config/metrics.py

```diff
@@ -39,6 +39,7 @@ class MetricsConfig(Config):
     def read_config(self, config, **kwargs):
         self.enable_metrics = config.get("enable_metrics", False)
         self.report_stats = config.get("report_stats", None)
+        self.report_stats_endpoint = config.get("report_stats_endpoint", None)
```

Comment:
wouldn't it make more sense to put the default here?

```suggestion
        self.report_stats_endpoint = config.get("report_stats_endpoint", "https://matrix.org/report-usage-stats/push")
```